libs/features/mas/src/price/utilities.js

```diff
@@ -331,24 +336,17 @@ const formatAnnualPrice = (data) => {
             return formatPrice(data, RecurrenceTerm.YEAR, getAnnualPrice);
         }
         const {
-            displaySummary: { outcomeType, duration, minProductQuantity = 1 } = {},
+            displaySummary: { outcomeType, duration } = {},
```

Comment:
this will break promos in case there are cases when promo object doesn't have minProductQuantity
I don't know if its a real use-case but it is easier to leave default then risking to break an existing feature

Reply:
I added default values for minProductQuantity and quantity in`isPromotionActive`, tested with usual promotion (back_2_school) and volume one, works for me